src/components/settings/PresetButtons.svelte

```diff
@@ -0,0 +1,56 @@
+<script>
+  import {
+    captionDuration,
+    captionFontSize,
+    videoSideSetting,
+    chatZoom,
+    livetlFontSize,
+    textDirection,
+    chatSplit,
+    autoVertical,
+    autoLaunchMode,
+    activePreset,
+    presets
+  } from '../../js/store';
+  import { importStores } from '../../js/storage';
+  import Button from 'smelte/src/components/Button';
+
+  $: active = $activePreset;
+  $: activePreset.set(active);
```

Comment:
Is it needed to bind the variable to the store this way for this component? I think we only used this technique in components in which we had to two-way bind to a store's value which sometimes doesn't work.